mappings/net/minecraft/entity/ai/brain/task/UnemployWhenJobSiteAbsentTask.mapping

```diff
@@ -0,0 +1 @@
+CLASS ako net/minecraft/entity/ai/brain/task/UnemployWhenJobSiteAbsentTask
```

Comment:
lose job on site loss task

Reply:
That's very good. I admit I didn't have a good name for that one, but I also wanted to not have the time wasted for figuering out what it's doing.
Generally better to not provide a name then, so other's time isn't wasted, I guess.